test/index-test.ts

```diff
@@ -13,17 +17,15 @@ import {
   tap,
   propagateEventTask
 } from '@most/core'
-import { newDefaultScheduler, delay, asap } from '@most/scheduler'
-import { hold } from '../src/index'
 
-const collect = (stream, scheduler) => {
-  const eventValues = []
+const collect = <A>(stream: Stream<A>, scheduler: Scheduler) => {
```

Comment:
Could we add return types to test helper function signatures?  I find it helps in reading/learning the tests.  For `collect`, the return type seems to be `Promise<readonly A[]>`

Reply:
fixed

Reply:
Thanks for adding all the return types.  Looks great.